src/AspNet.Security.OAuth.BattleNet/BattleNetAuthenticationOptions.cs

```diff
@@ -0,0 +1,63 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using Microsoft.AspNet.Authentication.OAuth;
+using Microsoft.AspNet.Http;
+using System;
+
+namespace AspNet.Security.OAuth.BattleNet {
+    /// <summary>
+    /// Defines a set of options used by <see cref="BattleNetAuthenticationHandler"/>.
+    /// </summary>
+    public class BattleNetAuthenticationOptions : OAuthAuthenticationOptions {
+        public BattleNetAuthenticationOptions() {
+            AuthenticationScheme = BattleNetAuthenticationDefaults.AuthenticationScheme;
+            Caption = BattleNetAuthenticationDefaults.Caption;
+            ClaimsIssuer = BattleNetAuthenticationDefaults.Issuer;
+
+            CallbackPath = new PathString(BattleNetAuthenticationDefaults.CallbackPath);
+
+			this.SetServerRegion(ServerRegion.US); //Default to US
+			
+            UserInformationEndpoint = BattleNetAuthenticationDefaults.UserInformationEndpoint;
+
+            SaveTokensAsClaims = false;
+        }
+
+		public void SetServerRegion(ServerRegion serverRegion)
```

Comment:
I'm not sure I really like this pattern. I'll think a bit more about it and get back to you ASAP! :+1: 

/cc @jerriep